src/components/selection-controls/Checkbox.js

```diff
@@ -12,7 +12,11 @@ export default {
   },
 
   props: {
-    indeterminate: Boolean
+    indeterminate: Boolean,
+    ripple: {
+      type: [Boolean, Object],
+      default: true
```

Comment:
This should be `default: () => ({ center: true })`

In the other files you used `default: { center: true }`, I don't know if this works too.